ecocodejava.properties

```diff
@@ -1,8 +1,8 @@
 category=External Analysers
 description=Provide a list of static code analyzers (for Java language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
 homepageUrl=https://github.com/green-code-initiative/ecoCode
```

Comment:
Doesn't this homepage need to change since the project has been moved?

Reply:
yes ... you're rigth ... but the old one is still active because this page will be the first entry point for our plugins.
but I'm going to update it

Reply:
done